libs/src/sdk/api/tracks/TracksApi.ts

```diff
@@ -181,6 +183,64 @@ export class TracksApi extends TracksApiWithoutStream {
     }
   }
 
+  /**
+   * Update a track
+   */
+  async updateTrack(
+    requestParameters: UpdateTrackRequest,
+    writeOptions?: WriteOptions
+  ) {
+    // Parse inputs
+    const { userId, trackId, coverArtFile, metadata, onProgress } =
+      parseRequestParameters(
+        'updateTrack',
+        createUpdateTrackSchema()
+      )(requestParameters)
+
+    // Upload track cover art to storage node
+    const coverArtResp = await retry3(
+      async () =>
+        await this.storage.uploadFile({
+          file: coverArtFile,
+          onProgress,
+          template: 'img_square'
+        }),
+      (e) => {
+        console.log('Retrying uploadTrackCoverArt', e)
+      }
+    )
+
```

Comment:
Looking at these lines from the `uploadTrack` method, should we add these adjustments to this method as well?
```
// Transform metadata
    const metadata = {
      ...parsedMetadata,
      ownerId: userId
    }
    const isPremium = metadata.isPremium
    const isUnlisted = metadata.isUnlisted
    // If track is premium, set remixes to false
    if (isPremium && metadata.fieldVisibility) {
      metadata.fieldVisibility.remixes = false
    }
    // If track is public, set required visibility fields to true
    if (!isUnlisted) {
      metadata.fieldVisibility = {
        ...metadata.fieldVisibility,
        genre: true,
        mood: true,
        tags: true,
        share: true,
        playCount: true
      }
    }
```